tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Serilog/LogsInjection/LoggerDispatchInstrumentation.cs

```diff
@@ -0,0 +1,78 @@
+// <copyright file="LoggerDispatchInstrumentation.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.Serilog.LogsInjection
+{
+    /// <summary>
+    /// LoggerFactoryScopeProvider.ForEach&lt;TState&gt; calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
```

Comment:
Yay we can safely support Serilog as far back as v1.4! 🎉 I'm guessing there weren't any good injection points in earlier versions of the library?

Reply:
It looks like this is present in assembly version `1.0.0.0` so on Monday I can update this, unless there's a reason not to go all the way back to 1.0.0.0

Reply:
After some research I found out that property names with underscores are only supported starting with 1.4.0.0, so I updated the integration definition to that assembly version and added a comment to persist that knowledge